src/ts/test/commands.spec.ts

```diff
@@ -34,5 +34,53 @@ describe("Commands", () => {
       const element = getByText(editorElement, "An sentence")
       expect(element.innerHTML).toBe("An <strong>improved</strong> sentence")
     });
+
+    it("should keep marks across the whole replaced text when suggestions are applied and additions are made to the beginning of the range", () => {
+      const match = createMatch(5, 9, [
+        { text: "beggar", type: "TEXT_SUGGESTION" }
+      ]);
+      const {
+        editorElement,
+        commands
+      } = createEditor("<p>Two <strong>eggs</strong></p>", [match]);
```

Comment:
`nit` It would be nice to see a test when some text is bold and italicised, to prove it can handle multiple marks.